motor/motor_gridfs.py

```diff
@@ -222,6 +222,7 @@ class AgnosticGridIn(object):
     readable = DelegateMethod()
     seekable = DelegateMethod()
     upload_date = ReadOnlyProperty()
+    md5 = ReadOnlyProperty()
```

Comment:
Let's not add this property since it's useless in motor.

Reply:
Ack, I'll find a way around this, might mean skipping a couple more tests.